tests/integration-tests/tests/dashboard_and_alarms/test_dashboard_and_alarms.py

```diff
@@ -0,0 +1,173 @@
+# Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License").
+# You may not use this file except in compliance with the License.
+# A copy of the License is located at
+#
+# http://aws.amazon.com/apache2.0/
+#
+# or in the "LICENSE.txt" file accompanying this file.
+# This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, express or implied.
+# See the License for the specific language governing permissions and limitations under the License.
+import datetime
+import math
+import time
+
+import boto3
+import pytest
+from assertpy import assert_that
+from botocore.exceptions import ClientError
+
+
+@pytest.mark.usefixtures("instance", "os", "scheduler")
+@pytest.mark.parametrize("dashboard_enabled, cw_log_enabled", [(True, True), (True, False), (False, False)])
+def test_dashboard_and_alarms(
+    dashboard_enabled,
+    cw_log_enabled,
+    region,
+    pcluster_config_reader,
+    clusters_factory,
+    test_datadir,
+):
+    cluster_config = pcluster_config_reader(
+        dashboard_enabled=str(dashboard_enabled).lower(),
+        cw_log_enabled=str(cw_log_enabled).lower(),
+    )
+    cluster = clusters_factory(cluster_config)
+    cw_client = boto3.client("cloudwatch", region_name=region)
+    headnode_instance_id = cluster.get_cluster_instance_ids(node_type="HeadNode")[0]
+    compute_instance_ids = cluster.get_cluster_instance_ids(node_type="Compute")
+
+    # test CWAgent metrics
+    # sleep for 2 minutes to ensure we can get metrics data
+    time.sleep(60 * 2)
+    start_timestamp, end_timestamp = _get_start_end_timestamp(minutes=2)
+    metrics_response_headnode = _get_metric_data(headnode_instance_id, cw_client, start_timestamp, end_timestamp)
+    mem_values = _get_metric_data_values(metrics_response_headnode, "mem")
+    disk_values = _get_metric_data_values(metrics_response_headnode, "disk")
+    assert_that(mem_values).is_not_empty()
+    assert_that(disk_values).is_not_empty()
+
+    if compute_instance_ids:
```

Comment:
How to make sure this is tested? What about we set min_count to 1 in the test? And here we can test if the scheduler is slurm, len(compute_instance_ids) >0

Reply:
After discussion, I set MinCount in the configuration file to make sure at least 1 compute node will be created so we can test one compute node (one will be enough).